pkgs/servers/tailscale/default.nix

```diff
@@ -15,7 +15,7 @@
 }:
 
 let
-  version = "1.72.1";
+  version = "1.74.0";
```

Comment:
Since tailscale 1.74 depends on go1.23, i think we should override it in all-packages.nix to use go_1_23